spark/src/main/scala/geotrellis/spark/resample/Implicits.scala

```diff
@@ -21,5 +21,6 @@ import geotrellis.spark._
 object Implicits extends Implicits
 
 trait Implicits {
-  implicit class withZoomResampleMethods[K: SpatialComponent](self: TileLayerRDD[K]) extends ZoomResampleMethods[K](self)
+  implicit class withTileLayerRDDZoomResampleMethods[K: SpatialComponent](self: TileLayerRDD[K]) extends TileLayerRDDZoomResampleMethods[K](self)
+  implicit class withMultibandTileLayerRDDZoomResampleMethods[K: SpatialComponent](self: MultibandTileLayerRDD[K]) extends MultibandTileLayerRDDZoomResampleMethods[K](self)
```

Comment:
You shouldn't need to add the specific method extensions of Multiband.
You should be able to add `V` type parameter constrained by `(implicit ev: V => TileResampleMethods[V])` to the `withZoomResampleMethods`/`ZoomResampleMethods` which should allow it to call `ZoomResample` directly.